<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PID_GUI</class>
 <widget class="QDialog" name="PID_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PID</string>
  </property>
  <widget class="QLabel" name="PID_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PID</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>146</width>
     <height>147</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="KiLabel">
      <property name="text">
       <string>Ki:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="Ki_SpinBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="Kp_SpinBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="KdLabel">
      <property name="text">
       <string>Kd:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="KpLabel">
      <property name="text">
       <string>Kp:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="Kd_SpinBox">
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tauLabel">
      <property name="text">
       <string>dt (ms):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="tau_SpinBox">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>30.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Set_PB">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SetAndClose_PB">
      <property name="text">
       <string>Set and close</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Close_PB">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="Plot" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SetAndClose_PB</sender>
   <signal>clicked()</signal>
   <receiver>PID_GUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close_PB</sender>
   <signal>clicked()</signal>
   <receiver>PID_GUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
